roadie.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 1.9"
 
-  s.add_dependency 'nokogiri', '~> 1.5'
+  s.add_dependency 'nokogiri', '~> 1.8.5'
```

Comment:
This locks the user to the 1.8.x branch instead of to the 1.x branch. I'd prefer `~> 1.8` so 1.9, 1.10, etc. are allowed.

Reply:
@Mange Sounds cool. 👍 I updated the PR